package.json

```diff
@@ -20,44 +20,46 @@
     "prettier:diff": "prettier --write --config ./.prettierrc --list-different 'src/**/*.ts'"
   },
   "lint-staged": {
-    "*.{ts,tsx}": [
+    "./src/**/*.{ts,tsx}": [
       "prettier --write --config ./.prettierrc --config-precedence file-override",
       "tslint --project .",
       "git add"
     ]
   },
   "peerDependencies": {
-    "typeorm": "^0.2.7"
+    "typeorm": "^0.3.0"
   },
   "devDependencies": {
-    "@types/express-session": "^1.15.11",
-    "@types/node": "^10.12.8",
-    "@types/webpack": "^4.4.19",
-    "@types/webpack-env": "^1.13.6",
-    "@types/webpack-merge": "^4.1.3",
-    "@types/webpack-node-externals": "^1.6.3",
+    "@types/express-session": "^1.17.5",
+    "@types/node": "^18.11.18",
+    "@types/webpack": "^5.0.0",
+    "@types/webpack-env": "^1.18.0",
+    "@types/webpack-merge": "^5.0.0",
+    "@types/webpack-node-externals": "^2.5.3",
     "awesome-typescript-loader": "^5.2.1",
-    "express-session": "^1.15.6",
-    "husky": "^1.1.4",
-    "lint-staged": "^8.0.4",
+    "husky": "^8.0.3",
+    "lint-staged": "^13.0.3",
     "prettier": "^1.15.2",
     "rimraf": "^2.6.2",
-    "ts-node": "^7.0.1",
+    "ts-node": "^10.7.0",
     "tslint": "^5.11.0",
     "tslint-config-prettier": "^1.16.0",
     "tslint-loader": "^3.5.4",
     "tslint-microsoft-contrib": "^5.2.1",
-    "typeorm": "^0.2.7",
+    "typeorm": "^0.3.0",
     "typescript": "^3.1.6",
-    "webpack": "^4.25.1",
-    "webpack-cli": "^3.1.2",
-    "webpack-merge": "^4.1.4",
-    "webpack-node-externals": "^1.7.2"
+    "webpack": "^5.75.0",
+    "webpack-cli": "^5.0.1",
+    "webpack-merge": "^5.8.0",
+    "webpack-node-externals": "^3.0.0"
   },
   "husky": {
     "hooks": {
       "post-commit": "git update-index --again",
       "pre-commit": "lint-staged"
     }
+  },
+  "dependencies": {
+    "express-session": "^1.17.3"
```

Comment:
Could this be a `peerDependency` instead?

Reply:
I followed your example for the typeorm package